### PR TITLE
chore: Replaces nodemon with node --watch

### LINKED ISSUE #5: Remove nodemon?
https://pawelgrzybek.com/til-node-v18-11-0-comes-with-a-watch-mode-so-you-might-not-need-nodemon/

### PR DESCRIPTION
**Pull request by davidwessman**
- Tested locally and seems to reload when files change
- Fixes #5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "type": "module",
   "scripts": {
-    "dev": "nodemon .",
+    "dev": "node --watch .",
```
Comment (raae) [inline package.json:9]: praise: perfect

**Review by raae — COMMENTED** [review]
I'll have to double check the node version on Railway before we merge.

**Comment by davidwessman** [comment]
@raae I think this one is only used in development, so it should not affect  Railway?

**Comment by raae** [comment]
🤦‍♀️

**Pull request merged**